ui/lib/sync/addon/components/secrets/landing-cta.hbs

```diff
@@ -6,11 +6,18 @@
 <SyncHeader @title="Secrets Sync">
   <:actions>
     {{#if this.version.isEnterprise}}
-      <Hds::Button @text="Create first destination" {{on "click" this.onCreate}} data-test-cta-button />
+      <Hds::Button
+        @text="Create first destination"
+        {{on "click" this.onCreate}}
+        data-test-cta-button
+        disabled={{@adapterError}}
+      />
     {{/if}}
   </:actions>
 </SyncHeader>
-
+{{#if @adapterError}}
+  <MessageError @errorMessage={{@adapterError.message}} />
```

Comment:
If you decide to go with my suggestion below then I think this should change to `@featureEnabled.message`

Reply:
Thank you!